src/ai/defaultai.cc

```diff
@@ -320,6 +341,11 @@ void DefaultAI::think() {
 		}
 	}
 
+	if (GameController* const ctrl = game().game_controller()) {
+		verb_log_dbg_time(gametime, "Jobs: %d; delay: %d; gamespeed: %d \n", jobs_to_run_count,
+		                  delay_time, ctrl->real_speed());
```

Comment:
I like this, what about preserving it? If condition it with 'if jobs_to_run_count > 3' to have this as warning?

Reply:
maybe we can just move it to our debug stats section Otherwise I would guard it with the same bool used there. 
I'll provide a proper solution to keep it for debugging.

Reply:
whatever looks good to you